### ISSUE TITLE
End-to-End Demand Response Model Identification and Baseline Estimation with Deep Learning. (arXiv:2109.00741v1 [eess.SY])

### ISSUE THREAD

**Issue by CoffeeKumazaki** [CoffeeKumazaki/arXiv#10511]
End-to-End Demand Response Model Identification and Baseline Estimation with Deep Learning. (arXiv:2109.00741v1 [eess.SY])<br>
https://ift.tt/3kPqpA3<br>
<p>This paper proposes a novel end-to-end deep learning framework that simultaneously identifies demand baselines and the incentive-based agent demand response model, from the net demand measurements and incentive signals. This learning framework is modularized as two modules: 1) the decision making process of a demand response participant is represented as a differentiable optimization layer, which takes the incentive signal as input and predicts user's response; 2) the baseline demand forecast is represented as a standard neural network model, which takes relevant features and predicts user's baseline demand. These two intermediate predictions are integrated, to form the net demand forecast. We then propose a gradient-descent approach that backpropagates the net demand forecast errors to update the weights of the agent model and the weights of baseline demand forecast, jointly. We demonstrate the effectiveness of our approach through computation experiments with synthetic demand response traces and a large-scale real world demand response dataset. Our results show that the approach accurately identifies the demand response model, even without any prior knowledge about the baseline demand.</p>
<br>
<br>
via eess.SY updates on arXiv.org http://arxiv.org/<br>
